testsuite/shared/src/main/java/org/jboss/as/test/shared/LayersTestBase.java

```diff
@@ -53,6 +53,7 @@ public abstract class LayersTestBase {
             "org.hornetq.client",
             // TODO we need to add an xts layer
             "org.jboss.as.xts",
+            "org.wildfly.extension.elytron.jaas-realm",
```

Comment:
This PR has needed more additions but in general could it be useful if the LatestTest got some of these items directly from a class in WildFly Core so when changes are made to WildFly Core we reduce the number of round trips to get something merged.